src/test/java/io/jenkins/plugins/analysis/core/graphs/NewVersusFixedSeriesBuilderTest.java

```diff
@@ -0,0 +1,32 @@
+package io.jenkins.plugins.analysis.core.graphs;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import io.jenkins.plugins.analysis.core.model.AnalysisResult;
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests class {@link NewVersusFixedSeriesBuilder}.
+ *
+ * @author Michaela Reitschuster
+ */
+class NewVersusFixedSeriesBuilderTest {
+
+    /**
+     * Verifies that computeSeries includes the totalSize and the fixedSize
+     * of an AnalysisResult.
+     */
+    @Test
+    void computeSeriesReturnsTotalAndFixedSize() {
```

Comment:
Typischerweise sollten Test Methoden mit should beginnen z.B. `shouldReturnNewAndFixedSize`.